pkg/util/version/const.go

```diff
@@ -18,14 +18,17 @@ var GitCommit = "unknown"
 
 // InstallStream describes stream we are defaulting to for all new clusters
 var InstallStream = &Stream{
-	Version:  NewVersion(4, 6, 26),
-	PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:e867135cd5a09192635b46ccab6ca7543e642378dc72fa22ea54961b05e322f2",
+	Version:  NewVersion(4, 7, 12),
+	PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:2029c5779202293f23418d47a1a823c4e4c8539c1ab25e8bda30d48335b4892e",
 }
 
 // Streams describes list of streams we support for upgrades
 var (
 	Streams = []*Stream{
-		InstallStream,
+		{
```

Comment:
Missing InstallStream, I think the 4.7 stream should still be included in Streams.

Reply:
Good catch! Updated.